<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Abhi1996" failures="2" tests="8" name="Test" time="102.648" errors="0" timestamp="2025-06-07T21:05:13 IST">
  <testcase classname="testCases.Tc_01_AccountRegistration" name="Verify_Account_registration" time="9.204"/>
  <testcase classname="testCases.TC_02_LoginToAccount" name="Verify_login" time="2.808"/>
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="2.645"/>
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="2.542"/>
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="21.726"/>
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="21.472">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_03_LoginDDT.VerifyLoginDDT(TC_03_LoginDDT.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="21.569"/>
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="2.949">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_03_LoginDDT.VerifyLoginDDT(TC_03_LoginDDT.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginDDT -->
</testsuite> <!-- Test -->
